apps/chef_db/src/chef_db.erl

```diff
@@ -230,6 +230,11 @@ fetch(ObjectRec, #context{reqid = ReqId}) ->
                       not_found |
                       {error, term()}.
 fetch_multi(RecModule, #context{reqid = ReqId}, QueryName, QueryParams) ->
+    %% FIXME: 
```

Comment:
That's actually kinda clever, but it does make me a little worried that it'd be inconsistent with everything else. For example, just a few lines above we do: `stats_hero:ctime(ReqId, {chef_sql, fetch},`. I guess it kinda comes down to how we envision ourselves using graphite. If we want to know more broadly "the service is slow because RDBMS requests are slow", we don't need a lot more granularity. If we want something more akin to a "slow query log", we'd want to include more info about what the query is. But maybe we pull that info from the request somehow?